dynamiq/nodes/writers/pgvector.py

```diff
@@ -0,0 +1,83 @@
+from typing import Any, Literal
+
+from dynamiq.connections import PGVector
+from dynamiq.nodes.node import NodeGroup, VectorStoreNode, ensure_config
+from dynamiq.runnables import RunnableConfig
+from dynamiq.storages.vector.base import BaseWriterVectorStoreParams
+from dynamiq.storages.vector.pgvector.pgvector import PGVectorStore
+from dynamiq.utils.logger import logger
+
+
+class PGVectorDocumentWriter(VectorStoreNode, BaseWriterVectorStoreParams):
```

Comment:
There need to add extra base class and use instead of BaseWriterVectorStoreParams:
https://github.com/dynamiq-ai/dynamiq/blob/9134b277128f2f59b2b9884ae726a787e8afaa1e/dynamiq/storages/vector/milvus/milvus.py#L16-L17

This automatically add `content_key` and `embedding_key` to node attributes. Currently `embedding_key` missed

Reply:
Thanks for the clarification, added the `PGVectorStoreParams` in the latest commit.

Reply:
Finished with the `PGVectorStoreParams` class, and updated it with relevant fields for the nodes.